src/Libraries/Microsoft.Extensions.AI/Functions/AIFunctionFactory.cs

```diff
@@ -32,6 +32,7 @@ public static class AIFunctionFactory
     public static AIFunction Create(Delegate method, AIFunctionFactoryCreateOptions options)
     {
         _ = Throw.IfNull(method);
+        _ = Throw.IfNull(options);
```

Comment:
Why don't we want to allow it to be null?

Reply:
Done.